libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/NavigationViewModel.java

```diff
@@ -215,11 +219,21 @@ private void initNavigationLocationEngine() {
   }
 
   private void initLanguage(NavigationUiOptions options) {
-    language = options.directionsRoute().routeOptions().language();
+    RouteOptions routeOptions = options.directionsRoute().routeOptions();
+    if (routeOptions != null) {
+      language = routeOptions.language();
+    } else {
+      language = localeUtils.getNonEmptyLanguage(getApplication(), null);
```

Comment:
We have [`LocaleUtils#inferDeviceLanguage(Context)`](https://github.com/mapbox/mapbox-navigation-android/blob/master/libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/utils/LocaleUtils.java#L38-L40) API available in `LocaleUtils` which does the same without having to pass `null`. What about using it instead of `localeUtils.getNonEmptyLanguage(getApplication(), null)`?